Rocket.Chat/Models/Auth.swift

```diff
@@ -40,7 +40,7 @@ final class Auth: Object {
 
     var user: User? {
         guard let userId = userId else { return nil }
-        return self.realm?.object(ofType: User.self, forPrimaryKey: userId)
+        return Realm.shared?.object(ofType: User.self, forPrimaryKey: userId)
```

Comment:
@cardoso Is there any reason to use `self.realm?` here?

Reply:
It was probably to avoid the Singleton pattern.
Was it causing the problem?